Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -153,12 +158,14 @@
 
     if bUseBasilQuantification
         % Here we perform FSL BASIL
-        PWI = xASL_quant_Basil(PWI, x);
+		if modules.asl.bMultiPLD
```

Comment:
Also, this can be made single- and multi-PLD ignorant. Input is the same, and output can be the same as well, `xASL_quant_Basil` should also initialize `ATT` and `Tex` as empty at the start of its function.

Reply:
sure. no prob.